app/langswitch/LangMessages.scala

```diff
@@ -222,7 +222,7 @@ object LangMessages {
 
   val `Repayment amount changed`: Message = Message("Repayment amount changed")
 
-  def `You claimed a VAT repayment of`(originalPostingAmount: String, vatToPay_Box5: String, url: String): Message = Message(s"You claimed £$originalPostingAmount. We calculated this amount was incorrect so we will repay you £$vatToPay_Box5.  This will reach your repayment bank account in 3 working days.  We sent you a letter explaining why we changed your amount. <br><br>If you do not receive a letter in the next 7 days, check your <a href=$url>VAT payments history</a>.")
+  def `You claimed a VAT repayment of`(originalPostingAmount: String, vatToPay_Box5: String, url: String): Message = Message(s"You claimed £$originalPostingAmount. We calculated this amount was incorrect so we will repay you £$vatToPay_Box5.  This will reach your repayment bank account in 3 working days.  We sent you a letter explaining why we changed your amount. </p><p>If you do not receive a letter in the next 7 days, check your <a href=$url>VAT payments history</a>.")
```

Comment:
This pattern of using fragments of `<p>` tags in the messages introduces a dependency on there being a corresponding opening/closing `<p>` tag in the view(s) that it's used in.

If you need <p> tags between bits of text, it's perhaps better to split the message up into two, have each message free of the the <p> tags, and keep the <p> tags in the view itself.